### PR TITLE
chore(maintenance): upgrade ruby, rails and dependencies to LTS (#ayuy7g)

### PR DESCRIPTION
**Pull request by DrRaider**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Create automatic tests
- [ ] No automatic tests failures
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

File: spec/dummy/config/application.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
+    # config.active_record.raise_in_transactional_callbacks = true
```
Comment (larcin) [inline spec/dummy/config/application.rb:23]: [MISC] Why did you comment this?

**Review by larcin — COMMENTED** [review]
Your commit message state that we upgraded ruby but it is not true

File: .travis.yml [hunk 2]
```diff
@@ -14,7 +14,8 @@ jobs:
       script: commitlint-travis
     - stage: Tests
       language: ruby
-      rvm: 2.3.4
+      before_install: gem install bundler -v '2.1.4'
+      rvm: 2.7.2
```
Comment (DrRaider) [inline .travis.yml:18]: @larcin we did upgrade ruby too, it's just not specified in gem file.

File: spec/dummy/config/application.rb (hunk 1 above)
Comment (DrRaider) [reply spec/dummy/config/application.rb:23]: It's deprecated in rails 5.1 and removed in rails 5.2. c.f: https://github.com/rails/rails/issues/35671#issuecomment-474627832

**Review by larcin — APPROVED** [review]

**Pull request merged**